code/game/world.dm

```diff
@@ -515,3 +516,8 @@ GLOBAL_VAR_INIT(failed_old_db_connections, 0)
     var/dll = world.GetConfig("env", "EXTOOLS_DLL")
     if (dll)
         call(dll, "debug_initialize")()
+
+
+/world/Del()
+	rustg_close_async_http_client() // Close the HTTP client. If you dont do this, youll get phantom threads which can crash DD from memory access violations
+	..()
```

Comment:
This *shouldnt* get me shot

Reply:
For context. `world/Reboot()` only triggers via the ingame reboot verb, waiting for round end or the hard reset in DD. Pressing the "Stop" button does not invoke `world/Reboot()`, so I cant use that as an area for thread cleanup.

`world/Del()` is called in all instances of the world deleting, apart from DD being forcefully quit (EG: From task manager, which will unload all the threads regardless if the parent process is killed)